package.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "the-wall-templater-dustjs",
 	"description": "The Wall compatible template handler",
-	"version": "2.4.1",
+	"version": "2.5.0",
```

Comment:
as far as I can tell this change is not backwards compatible?  The constructor has changed arguments.  If so, we need a major revision bump.